enhancements/machine-config/mco-cgroupsv2-support.md

```diff
@@ -0,0 +1,192 @@
+---
+title: Control Group v2 Enablement
+authors:
+  - "@rphillips"
+reviewers:
+  - "@mrunalp"
+approvers:
+  - "@yuqi-zhang"
+  - "@mrunalp"
+creation-date: 2021-10-19
+last-updated: 2021-10-19
+status: implementable
+---
+
+# Control Group v2 Enablement
+
+
+## Release Signoff Checklist
+
+- [ ] Enhancement is `implementable`
+- [ ] Design details are appropriately documented from clear requirements
+- [ ] Test plan is defined
+- [ ] Operational readiness criteria is defined
+- [ ] Graduation criteria for dev preview, tech preview, GA
+- [ ] User-facing documentation is created in [openshift-docs](https://github.com/openshift/openshift-docs/)
+
+## Definitions and References
+
+- cgroup: control group and is never capitalized
+- cgroups: multiple control cgroups
+- cgroup v1: references cgroup version 1 implementation
+- cgroup v2: references cgroup version 2 implementation
+
+[Upstream Docs](https://www.kernel.org/doc/html/latest/admin-guide/cgroup-v2.html)
+
+## Summary
+
+Control Group v2 (cgroup v2) enablement in Kubernetes has progressed to the
+point that it is usable. The underlying runtime (cri-o) and supporting
+subsystems are now ready for customers to begin their own testing with it. Not
+all workloads will be compatible with cgroup v2, so it will *not* be enabled by
+default within OpenShift at this time.
+
+## Motivation
+
+Migrating to cgroup v2 will bring in many new features and fixes not found in
+cgroup v1. cgroup v1 is considered 'legacy' and migrating to cgroup v2 is
+considered necessary since RHEL ships with cgroup v2 on by default. (OpenShift
```

Comment:
s/RHEL/RHEL9/

(Also of note related to this, current Fedora defaults to v2, and I believe OKD is still reverting that - hopefully we can also validate this flag on OKD)

When we're thinking about RHEL9 - would one expect that upgrading to that automatically inherits the OS default to cgroup v2, or would the admin need to flip on this flag in addition?  I would hope the former.

Reply:
The intent is to enable this by flag at install time.